ibm/resource_ibm_satellite_location.go

```diff
@@ -299,8 +299,8 @@ func resourceIBMSatelliteLocationRead(d *schema.ResourceData, meta interface{})
 		d.Set(sateLocZone, *instance.Datacenter)
 	}
 
-	if instance.WorkerZones != nil {
-		d.Set("zones", instance.WorkerZones)
+	if zones, ok := d.GetOk("zones"); ok {
```

Comment:
same here

Reply:
Removed GetOk method and set the zones parameter in the imported section.